colors/dracula.vim

```diff
@@ -237,10 +241,15 @@ call s:h('Normal', s:fg, s:Background())
 call s:h('StatusLine', s:none, s:bglighter, [s:attrs.bold])
 call s:h('StatusLineNC', s:none, s:bglight)
 call s:h('WildMenu', s:bg, s:purple, [s:attrs.bold])
-call s:h('CursorLine', s:none, s:subtle)
 
-hi! link ColorColumn  DraculaSelection
-hi! link CursorColumn DraculaSelection
+if g:dracula_cursorline
+  call s:h('CursorLine', s:none, s:subtle)
+else
+  hi clear CursorLine
+endif
+
+hi! link ColorColumn  DraculaBgDark
+hi! link CursorColumn DraculaBgDark
```

Comment:
`ColorColumn` is indeed subtle (👍 ); I think `CursorColumn` is a bit _too_ subtle (I had to search hard to find it).

Reply:
Suggestions?

Reply:
I'm not sure. The default red seems ugly. I just don't use it to know what to look for.

Reply:
It's showing up red for you?

Reply:
My vim -u NONE is the same color as my terminal foreground color.

I don't use it either, but I can't imagine something as noticable as the foreground color would be very useful in any situation. The dark on my terminal is not difficult to see at all, which makes me wonder why it is on yours?

Reply:
Sorry I got mixed up between cursor column and color column. Ignore the red.

The dark on my terminal can be difficult to see because I use our Terminal.app theme as well, so the background is already quite dark.